src/unauthenticated/index.tsx

```diff
@@ -20,6 +24,8 @@ type Props = {
 
 const UnauthenticatedRoot = ({ firstTimeUser }: Props) => {
   const initialRouteName = firstTimeUser ? '/landing' : '/signin'
+  const onboarded = useAppSelector((state) => state.user?.user?.onboarded)
+  const isOnboarded = onboarded ? '/signin' : '/onboarding'
```

Comment:
Did you forget to remove this?  Seems like it is unused.

Reply:
Yes, I must've missed this when looking at the diff